src/commands/hosting-channel-deploy.ts

```diff
@@ -127,20 +142,31 @@ export default new Command("hosting:channel:deploy [channelId]")
         })
       );
 
-      await deploy(["hosting"], options, { hostingChannel: channelId });
+      const results = (await deploy(["hosting"], options, {
+        hostingChannel: channelId,
+      })) as DeployResult;
```

Comment:
My TS-brain is rusty: could this be `const results: DeployResult = await deploy()` rather than using `as`?

Reply:
The method `deploy()` returns `any` since it is implemented in js. Without the `as` cast, eslint issues this warning: `Unsafe assignment of an any value.`.

I'm fine with either construct. Please advise.